src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,112 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private int size;
+    private int threshold;
+    private List<Entry<K, V>>[] buckets;
```

Comment:
Using raw types in Java generics is not recommended. Use the diamond operator to maintain type safety. Change the declaration to 'List<Entry<K, V>>[] buckets;'